Feature/jk3server

AlexCS1337 opened the pull request:
# Description

This adds Jedi Knight: Jedi Academy configuration to LinuxGSM
Based off jk2server configuration with a few differences.
Also updated the port used in the jk2server configuration because it was using the wrong one.

Fixes #2015

## Type of change

* [ ] Bug fix (a change which fixes an issue).
* [ ] New feature (change which adds functionality).
* [* ] New Server (new server added).
* [* ] Refactor (restructures existing code).
* [ ] Comment update (typo, spelling, explanation, examples, etc).

## Checklist

PR will not be merged until all steps are complete.

* [ ] This pull request links to an issue.
* [ ] This pull request uses the `develop` branch as its base.
* [* ] This pull request Subject follows the Conventional Commits standard.
* [* ] This code follows the style guidelines of this project.
* [* ] I have performed a self-review of my code.
* [ ] I have checked that this code is commented where required.
* [* ] I have provided a detailed with enough description of this PR.
* [ ] I have checked If documentation needs updating.

## Documentation

If documentation does need updating either update it by creating a PR (preferred) or request a documentation update.
* User docs: https://github.com/GameServerManagers/LinuxGSM-Docs
* Dev docs: https://github.com/GameServerManagers/LinuxGSM-Dev-Docs

**Thank you for your Pull Request!**

Commit:
Update port number for jk2server

Commit:
Add jk3server LinuxGSM configuration
This adds Jedi Knight: Jedi Academy configuation to LinuxGSM
Similar to jk2server configuration but with minor differences.

Commit:
Update port number and defaultmap for jk3server

AlexCSilva commented:
Oops forgot to edit some other files will do that tomorrow

AlexCS1337 committed:
feat(new server): Jedi Knight: Jedi Academy

AlexCS1337 committed:
fix(jk3): misplaced number

dgibbs64 commented:
https://github.com/GameServerManagers/LinuxGSM/issues/3467

dgibbs64 commented:
Currently this pr is missing the server files of a mechanism to install the server. I have seen using openjk might be a good idea

AlexCS1337 committed:
Merge and fix conflicts

AlexCS1337 committed:
Update serverlist.csv

Try to add new server to serverlist

AlexCS1337 committed:
add update jk3 module
also added jk3 to serverlistmenu

AlexCS1337 committed:
update jk3s lgsm config

AlexCS1337 commented:
@dgibbs64 I've updated and tested recently and it works somewhat but still missing the server binary, added an update script for it but not sure how to get it working, the link for openjk is at https://github.com/JACoders/OpenJK/releases

AlexCS1337 committed:
remove serverlistmenu.csv
not meant to be there

AlexCS1337 committed:
Update missing info for lgsm modules - jk3server

AlexCS1337 committed:
fix typos

AlexCS1337 committed:
add condition for starting jk3server in tmux
